.env.template

```diff
@@ -0,0 +1,6 @@
+AERIE_USERNAME=
+AERIE_PASSWORD=
+
+POSTGRES_DB=
+POSTGRES_USERNAME=
```

Comment:
I'm wondering if we should call this `POSTGRES_USER` instead of `POSTGRES_USERNAME` to align with the same env var `postgres` actually uses?

Reply:
I initially set it to `USERNAME` instead of `USER` so it was more obvious that this was a second, separate set of credentials, completely independent from the `AERIE` credentials. If you feel like this won't be an issue then I've got no problems changing it, although if you change it in the Github environment let me know so I can modify `test.yml` accordingly.

Reply:
Is the assumption that we will always pass the `POSTGRES_USERNAME` env to the `POSTGRES_USER` env in the `postgres` container?

Reply:
Sort of. `POSTGRES_USERNAME` is only used in the `postgres` container's `POSTGRES_USER` env. However, `POSTGRES_USER` is also an env var used by the `aerie-gateway` container, this time with a value of `AERIE_USERNAME`.

Reply:
Ah yeah that is a little confusing. I think that is more of an issue with the name of the gateway env var tho. We should probably rename at some point. 
I think since `POSTGRES_USERNAME` in the `.env` is really meant to define the `POSTGRES_USER` for the `postgres` container we should probably just call it `POSTGRES_USER` to reflect that.

Reply:
Will do then. Again, just to clarify, will you be updating the name in e2e-test as well?

Reply:
Yup will do it now!

Reply:
Done!